CRM/Utils/Check/Component/Env.php

```diff
@@ -262,10 +262,9 @@ public function checkDefaultMailbox() {
         \Psr\Log\LogLevel::WARNING,
         'fa-envelope'
       );
-      $docUrl = 'target="_blank" href="' . CRM_Utils_System::docURL(['page' => 'user/advanced-configuration/email-system-configuration/', 'URLonly' => TRUE]) . '""';
       $message->addHelp(
         ts('A default mailbox must be configured for email bounce processing.') . '<br />' .
-        ts("Learn more in the <a %1>online documentation</a>.", [1 => $docUrl])
+          CRM_Utils_System::docURL2('user/advanced-configuration/email-system-configuration/')
```

Comment:
Are we loosing the original link text here?

Reply:
Yes ... in some places we have things like this one: "Learn more in the online documentation" with a link, other places "See the XXX section in the documentation" with a link, others use the default rendering of "(Learn more ...)" with a link.  I've adopted the more concise, default form. 

So this one changes from:
"A default mailbox must be configured for email bounce processing. Learn more in the <a>online documentation</a>."
to
 "A default mailbox must be configured for email bounce processing. <a>(Learn more...)</a>"

Reply:
I prefer the more concise form.

Reply:
However I can't see where the "Learn more..." text is being passed in here.

Reply:
Nope, I see it - in the `docUrl` functions.

Reply:
It's generated by docURL2()